wandb/sdk/lib/printer.py

```diff
@@ -191,7 +191,14 @@ def link(self, link: str, text: Optional[str] = None) -> str:
     def emoji(self, name: str) -> str:
         emojis = dict()
         if platform.system() != "Windows" and wandb.util.is_unicode_safe(sys.stdout):
-            emojis = dict(star="⭐️", broom="🧹", rocket="🚀", gorilla="🦍", turtle="🐢")
+            emojis = dict(
+                star="⭐️",
+                broom="🧹",
+                rocket="🚀",
+                gorilla="🦍",
+                turtle="🐢",
+                lightning="️⚡",
```

Comment:
nit: lightning weird again?

Reply:
I think this is just something with how that character is displayed in github's code view. If you copy and paste that character elsewhere it looks like ⚡